kyo-kernel/shared/src/main/scala/kyo/kernel/Effect.scala

```diff
@@ -61,7 +61,11 @@ object Effect:
         end try
     end catching
 
-    private[kyo] def defer[A, S](f: Safepoint ?=> A < S)(using _frame: Frame): A < S =
+    private[kyo] def defer[A, S](f: Safepoint ?=> A < S)(using Frame): A < S =
```

Comment:
It's not possible to inline defer directly because it's used by `Safepoint` and it would increase the size of the kernel's inlined bytecode. I've created a separate method